docs/.vitepress/theme-switcher-listener.js

```diff
@@ -2,10 +2,10 @@ document.addEventListener('change', () => {
   if (window.theme) {
     const stylesheets = document.querySelectorAll('link[rel="stylesheet"]');
     stylesheets.forEach((s) => {
-      if (s.getAttribute('href').includes('@warp-ds/tokens')) {
+      if (s.getAttribute('href').includes('@warp-ds/css')) {
         s.setAttribute(
           'href',
-          `https://assets.finn.no/pkg/@warp-ds/tokens/v1/${theme}.css`,
+          `https://assets.finn.no/pkg/@warp-ds/css/v1/tokens/${theme}.css`,
```

Comment:
Since the theme switching logic was previously only replacing this one href with brand CSS, and that doesn't include fonts, I believe we could keep the logic but also load all font setups in the docs head (for Tori, Blocket, Finn). Having all fonts defined this way shouldn't cause any trouble because the font that is picked up is the one that is actually used like e.g. body, :host { font-family: "Finntype"; } (based on the brand CSS).